internal/cmd/instance/get_instance.go

```diff
@@ -56,13 +55,9 @@ func (gb *GetInstanceCmd) Run() error {
 		return nil
 	}
 
-	resultInstances := &types.ServiceInstances{}
+	resultInstances := &types.ServiceInstances{Vertical: true}
```

Comment:
Can we use `types.ServiceInstance` and remove the Vertical property of the plural type?

Reply:
There is a case, where you can have more than one instance with the same name. However this cases are rare and if there is such a situation the instances probably wouldn't be too much. That's why decided to make it vertical, since most of the time the instance will be just one and the readability would be significantly improved.

Reply:
If there 2 instances and it looks good then it is ok for me